api/src/main/java/com/debugmafia/clueless/model/BoardLocationType.java

```diff
@@ -0,0 +1,6 @@
+package com.debugmafia.clueless.model;
+
+  enum BoardLocationType {
+    ROOM, HALLWAY
+  };
```

Comment:
I think this has to be `public enum BoardLocationType`? Otherwise callers won't be able to access this enum (it's private to this scope). Also the padding to the left can be removed.